<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortWidget</class>
 <widget class="QWidget" name="SerialPortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxTest">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>串口测试</string>
   </property>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecv">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRecv">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>33</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSend">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>130</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>381</width>
     <height>212</height>
    </rect>
   </property>
   <property name="title">
    <string>串口参数</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>     端口选择:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxComPort">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>       波特率:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBaudrate">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>       数据位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDataBit">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>     奇偶校验:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxParity">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>       停止位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStopBit">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>170</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMacthionAddr">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWorkKind">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>11</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设备地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工作模式：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
